app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -475,10 +404,10 @@ def emit_middleware_notification(type, op_name, op_arg, ems_ref, klass)
                   end
 
       Notification.create(:type => type, :options => {
-        :op_name   => op_name,
-        :op_arg    => op_arg.nil? ? '' : op_arg,
-        :mw_server => "#{mw_server.name} (#{mw_server.feed})"
-      })
+                            :op_name   => op_name,
+                            :op_arg    => op_arg.nil? ? '' : op_arg,
+                            :mw_server => "#{mw_server.name} (#{mw_server.feed})"
+                          })
```

Comment:
Yes, indented it that way because rubocop was complaining. But yours looks much better. Using it.